src/apps/projinfo.cpp

```diff
@@ -128,6 +129,13 @@ static void usage() {
                  "by '-' to disable them"
               << std::endl;
     std::cerr << std::endl;
+    std::cerr << "list-crs-filter is a comma separated combination of: "
+                 "auth_name=XXX,allow_deprecated,geodetic,geocentric,"
+              << std::endl;
+    std::cerr << "geographic,geographic_2d,geographic_3d,vertical,"
```

Comment:
you can remove temporal,engineering,bound,other : none of them can be stored in the DB currently. We can easily add them later. Removing them will make the doc clearer.

Reply:
done